docs/release_guide.md

```diff
@@ -2,31 +2,31 @@ In order to release a new version of `demisto-sdk` to the public follow these st
-7) Open a PR for that content-private branch, and verify the build triggered is green. Note that **this PR is for SDK version check only and it shouldn't be merged**, after build is finished successfully - you should discard this change and close the PR.
+7) Open a PR for that content-private branch, and verify the build triggered is green. Note that **this PR is for SDK version check only, and it shouldn't be merged**, after build is finished successfully - you should discard this change and close the PR.
 8) Create a new content branch and update the version of the SDK in Demisto's Content repository by updating the demisto-sdk version in the [**dev-requirements-py3.txt**](https://github.com/demisto/content/blob/master/dev-requirements-py3.txt) file. Use the release branch first - replace the `demisto-sdk==version` line with this line: `git+https://github.com/demisto/demisto-sdk.git@release-branch-name.`
 9) Open a PR for that content branch, and verify that the build triggered is green. Note that in order to trigger the build, opening the PR is required.
 
 ### Release process:
-1) In demisto-sdk repository's main page click on **releases**.
-2) Click on **Draft a new release**.
-3) Update the **Tag version** and **Release title** - the form should be `vX.X.X` .
-4) In the **Describe the release** text box enter the CHANGELOG contents for this release.
-5) If nightly passes ok, press **Publish release** - your release is now public.
-6) Update the [**dev-requirements-py3.txt**](https://github.com/demisto/content/blob/master/dev-requirements-py3.txt) file again, this time with the release tag.
-7) Run the regular build again or force merge your PR to the Content repository.
-8) Announce regarding the SDK release in the content-team slack channel.
-9) Update **CHANGELOG.md** file - change the `# Changelog` header to the release version in the format `# X.X.X` e.g. `# 1.0.0`, and create a new `# Changelog` header at the top of the file.
+1) Click [Here](https://github.com/demisto/demisto-sdk/releases/new) (alternatively: visit the [SDK github page](https://github.com/demisto/demisto-sdk), click on **releases**, and then **Draft a new release**)
+2) Update the **Tag version** and **Release title** to `vX.X.X`.
+3) In the **Describe the release** text box paste the `CHANGELOG` contents for this release.
+4) Make sure the relevant nightly SDK build passed (step 3 on the previous section), then click **Publish release**. Your release is now public 🎉.
+5) Update [**dev-requirements-py3.txt**](https://github.com/demisto/content/blob/master/dev-requirements-py3.txt) again, this time with the newly-released version (rather than the branch).
```

Comment:
```suggestion
5) Update [**dev-requirements-py3.txt**](https://github.com/demisto/content/blob/master/dev-requirements-py3.txt) again, this time with the newly-released version (rather than the branch), e.g demisto-sdk==x.x.x.
```